packages/deployment/scripts/integration-test.sh

```diff
@@ -23,17 +33,15 @@ export AG_SETUP_COSMOS_HOME=${AG_SETUP_COSMOS_HOME-$PWD}
 export AG_SETUP_COSMOS_STATE_SYNC_INTERVAL=20 
 AGORIC_SDK_PATH=${AGORIC_SDK_PATH-$(cd "$thisdir/../../.." > /dev/null && pwd -P)}
 
-if [ -d /usr/src/testnet-load-generator ]
+if [ -n "$LOADGEN" ]
 then
-  solodir=/usr/src/testnet-load-generator/_agstate/agoric-servers/testnet-8000
+  solodir="$LOADGEN"/_agstate/agoric-servers/testnet-8000
   "$thisdir/../../solo/bin/ag-solo" init "$solodir" --webport=8000
   SOLO_ADDR=$(cat "$solodir/ag-cosmos-helper-address")
   VAT_CONFIG="@agoric/vats/decentral-demo-config.json"
 fi
 
-# Speed up the docker deployment by pre-mounting /usr/src/agoric-sdk.
-DOCKER_VOLUMES="$AGORIC_SDK_PATH:/usr/src/agoric-sdk" \
```

Comment:
I removed it from this script because I wanted to be able to run the `integration-test.sh` without mounting my local SDK checkout inside the containers (i.e. use the Docker image's SDK instead).

In the most recent changes, I moved this setting up into the CI test and `run-deployment-integration.sh` to reenable your workflow.